### DISCUSSION TITLE
Typing animation in homepage

### DISCUSSION THREAD

**Discussion by chaoliu-cl** [nunocoracao/blowfish#848] [Q&A] [answered]
Dear Blowfish developers,

Thank you for the wonderful theme. I am wondering if it is possible to add a typing animation in the homepage.

Best,

Chao

**Comment by nunocoracao** [comment, marked as answer]
Check how to add a index.md to the homepage and then have a look at the TypeIt shortcode - https://blowfish.page/docs/shortcodes/#typeit
